### ISSUE TITLE
The Tribe Page

### ISSUE THREAD

**Issue by katjam** [neontribe/www#71]
**What needs doing?**
We'd like to display 3 words as well as name on front of person cards.

**Plan**
- [x] Design to incorporate space for 3 words per person card
- [ ] Get 3 words from everyon (can be recarded) - we also need images.

**Comment by HedgehogHolly** [comment]
![Screenshot 2019-05-14 at 10 56 56](https://user-images.githubusercontent.com/41140803/57689208-045d1a00-7637-11e9-95bf-a024ed34f326.png)

**Comment by HedgehogHolly** [comment]
Discussions from Catch up meeting 14th May #76 

**We like the new design but are concerned about:**
- it looking too cluttered/overwhelming with everyone (18 people) having 3 words
- what happens if the word is really long

**Actions**
- experiment more with the format
- look into wether hover/focus effects could help

**Comment by katjam** [comment]
Looking good. Hover / focus won't help too much on touchscreen devices & I think looks OK so far - Good idea though to keep an eye - 18 people x 3 words + name is a lot of clutter.

**Comment by HedgehogHolly** [comment]
**Again, talking with David I've now come up with this design** 

![Screenshot 2019-05-16 at 15 14 31](https://user-images.githubusercontent.com/41140803/57860761-534cb000-77ed-11e9-97e0-adee91a3de36.png)

**Having all black and white photos is less distraction and its less likely that the photo will dominate over the name of the person. In previous designs you tended to see the photos first.**

**Hoping to stick to three words per person but If the words are particularly long I would imagine it looking like this, where it falls onto two lines and the white area takes up more of the image**

![Screenshot 2019-05-16 at 15 26 09](https://user-images.githubusercontent.com/41140803/57861681-f4883600-77ee-11e9-924c-2f7afc26271c.png)

**Comment by katjam** [comment]
I like it.

**Comment by HedgehogHolly** [comment]
I've coded the design, so if its approved it's all ready to go. I've made the images greyscale and fade to colour on hover/focus. The only thing I haven't been able to do is alternate the colours for the strip, they are currently all pink. 

![Screenshot 2019-05-17 at 11 04 07](https://user-images.githubusercontent.com/41140803/57929247-ac2e4e00-78aa-11e9-999e-a76dd28de430.png)

https://github.com/neontribe/www/tree/71-three-words
